EU4toV2/Data_Files/configurables/country_mappings.txt

```diff
@@ -1533,6 +1533,9 @@ link = { eu4 = k_angleland v2 = AES } # Anglo-Saxony
 link = { eu4 = k_frankish v2 = FRK } # Frankia
 link = { eu4 = k_suebia v2 = SUE } # Suebi
 
+## Fraticelli Pope
+link = { eu4 = ASY v2 = ASY } # Assyria
```

Comment:
This doesn't look like a Fraticelli Pope mapping.

Reply:
In the CK2 to EU4 converter, it lists Assyria as a tag in the special_countries.txt under Fraticelli Pope.